azure_billing/scrape.py

```diff
@@ -63,7 +63,7 @@ def extract_metrics_from_df(df, counter):
         counter.record(value.ExtendedCost, **meta)
 
 
-def data(enrollment, token, month=None):
+def data(enrollment, token, metric_name, month=None):
```

Comment:
`data()` is not really a good name for this function. It does not contain a verb/action, and it is totally generic as to what kind of data would be returned.